src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
```

Comment:
- [ ] Your `App.js` is supposed to be your **main component** that would house every other components in your application. It should not be removed. Kindly restore it and import your `Calculator component` inside it.

![project-rqt-1](https://user-images.githubusercontent.com/26597432/226591320-2fe0b1c7-c6a1-4438-a11f-6a858171c216.png)